src/webui/index.js

```diff
@@ -143,6 +149,10 @@ module.exports = async function () {
   url.searchParams.set('deviceId', await ctx.getProp('countlyDeviceId'))
 
   ctx.setProp('launchWebUI', async (path, { focus = true, forceRefresh = false } = {}) => {
+    if (window.isDestroyed()) {
+      logger.error(`[web ui] window is destroyed, not launching web ui with ${path}`)
+      return
+    }
```

Comment:
e2e tests trying to launch webui even though it already killed electron and is done with the tests.